src/Bridges/bridge_optimizer.jl

```diff
@@ -1149,7 +1149,12 @@ function MOI.supports(
     attr::MOI.AbstractVariableAttribute,
     ::Type{MOI.VariableIndex},
 )
-    return MOI.supports(b.model, attr, MOI.VariableIndex)
+    if !MOI.supports(b.model, attr, MOI.VariableIndex)
```

Comment:
We `supports`, we prefer false positive than false negative. Previously, there were false positive but here, there might be false negative in case not all variables are set.